src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -199,11 +198,9 @@ private bool IsNotAbsoluteUri
         //
         // Checks if Iri parsing is allowed by the syntax & by config
         //
-        private bool _iriParsing;
+        private bool IriParsing => IriParsingStatic(_syntax);
 
-        //
-        // Statically checks if Iri parsing is allowed by the syntax & by config
-        //
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
It wasn't otherwise getting inlined?  What about InFact and the methods it calls?

Reply:
I checked and thankfully it was, same for all flags check helper methods

Reply:
Then why add the attribute?